jbi/services.py

```diff
@@ -1,15 +1,19 @@
 """Services and functions that can be used to create custom actions"""
+from __future__ import annotations
+
 import concurrent.futures
 import logging
-from typing import Dict, List
+from typing import TYPE_CHECKING, Dict, List
 
 import backoff
 import bugzilla as rh_bugzilla
 from atlassian import Jira, errors
 from statsd.defaults.env import statsd
 
 from jbi import environment
-from jbi.models import Actions
+
+if TYPE_CHECKING:
+    from jbi.models import Actions
```

Comment:
Only import on `TYPE_CHECKING`? (What value is TYPE_CHECKING? are we expecting this as False?)